campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -0,0 +1,321 @@
+            {{ button_label }}
+          </a>
+        </div>
+      {% endif %}
+  {% if blockable %}
+    </div>
+
+    <button class="block-snippet-button" title="{{ block_button_text|default('Remove this', true) }}">
+    </button>
+  {% endif %}
+<script type="text/javascript">
+  //<![CDATA[
+  (function() {
+    var snippet = document.getElementById('mofo-eoy-fundraiser-{{ snippet_id }}');
+
+    snippet.addEventListener('show_snippet', function () {
+      setTimeout( function() {
+        {% if donation_amount_first %}
+
+          var btnAmounts = snippet.querySelectorAll('.donation-amount');
+          var btnSelected;
+          var amount;
+          var frequency = 'single';
+
+          var locale = '{{ locale|default("en-US", true) }}';
+          var currencyCode = '{{ currency_code|default('usd', true) }}';
+          var btnDonate = snippet.querySelector('.button-link');
+
+          var donationURL = btnDonate.attributes['href'].value;
+          var delimiter = "?";
+          if (donationURL.indexOf('?') >= 0) {
+            delimiter = '&';
+          }
+          var dAmountFirst = '{{donation_amount_first}}';
+          var dAmountSecond = '{{donation_amount_second}}';
+          var dAmountThird = '{{donation_amount_third}}';
+          var dAmountFourth = '{{donation_amount_fourth}}';
+          var presets = [dAmountFirst, dAmountSecond, dAmountThird, dAmountFourth];
+
+          var numberFormat = new Intl.NumberFormat(locale, {
+            style: 'currency',
+            currency: currencyCode,
+            minimumFractionDigits: 0
+          });
+          var dAmountFirstEl = snippet.querySelector('.donation-amount-first');
+          var dAmountSecondEl = snippet.querySelector('.donation-amount-second');
+          var dAmountThirdEl = snippet.querySelector('.donation-amount-third');
+          var dAmountFourthEl = snippet.querySelector('.donation-amount-fourth');
+
+          dAmountFirstEl.textContent = numberFormat.format(dAmountFirst);
+          dAmountSecondEl.textContent = numberFormat.format(dAmountSecond);
+          dAmountThirdEl.textContent = numberFormat.format(dAmountThird);
+          dAmountFourthEl.textContent = numberFormat.format(dAmountFourth);
+
+          function selectAmount(element) {
+            if (!element) {
+              return;
+            }
+            element.classList.add('button-selected');
+            if (btnSelected) {
+              btnSelected.classList.remove('button-selected');
+            }
+            btnSelected = element;
+            amount = element.attributes['data-amount'].value;
+            updateLink();
+          }
+          function updateLink() {
+            btnDonate.attributes['href'].value = donationURL + delimiter + 'amount=' + amount + "&frequency=" + frequency + "&currency=" + currencyCode + '&presets=' + presets;
+          }
+          var selected = "{{ selected_button|default("donation-amount-second", true) }}";
```

Comment:
Or the other way around. You mix a lot `'` with `"`, please stick to one code style :)